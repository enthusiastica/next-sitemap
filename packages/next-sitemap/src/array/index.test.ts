import { toChunks } from '.'

describe('next-sitemap/array', () => {
  test('toChunks', () => {
    const inputArray = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
    const chunkSize = 3

    const chunks = toChunks(inputArray, chunkSize)

    expect(chunks).toMatchSnapshot()
    expect(chunks.length).toBe(Math.ceil(inputArray.length / chunkSize))
  })
})
