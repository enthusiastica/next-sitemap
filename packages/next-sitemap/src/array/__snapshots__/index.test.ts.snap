// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`next-sitemap/array toChunks 1`] = `
Array [
  Array [
    0,
    1,
    2,
  ],
  Array [
    3,
    4,
    5,
  ],
  Array [
    6,
    7,
    8,
  ],
  Array [
    9,
    10,
  ],
]
`;
